solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -173,13 +174,19 @@ contract RandomBeacon is Ownable {
 
     event RelayEntrySubmitted(uint256 indexed requestId, bytes entry);
 
+    event RelayEntryTimedOut(uint256 indexed requestId);
```

Comment:
The report timeout action will cause a retry if of the relay request thus will emit a new `RelayEntryRequested` with the same request ID but new `groupId`. I don't think we need to add it here.

Reply:
In case of request retry we emit a new `RelayEntryRequested` event (see `retryOnEntryTimeout` introduced by https://github.com/keep-network/keep-core/pull/2664/commits/e357f7d1b0266e7449edfcb7bc69e7af58dda7bf) so the new group is aware. I don't think we need to put `groupId` here.

Reply:
Just to be clear, the `groupId` we are talking about here means "the group ID of the terminated group" not the "group ID of the group selected for retry" right? If so, maybe let's call it `terminatedGroupId` to avoid confusion we had in beacon v1?

Reply:
Yep! The one for which the entry timed out. I think `terminatedGroupId` is 👌🏻

Reply:
Addressed in https://github.com/keep-network/keep-core/pull/2671/commits/a8a0882231843158d659a7b75e3b209ccf8ef092 part to of the follow-up PR: https://github.com/keep-network/keep-core/pull/2671